plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java

```diff
@@ -233,6 +239,29 @@ public void onEnable() {
             return;
         }
 
+        try {
+            this.adventure = BukkitAudiences.create(this);
+        }
+        catch (Exception exception) {
+            logger.error("Could not initialize adventure platform", exception);
+            this.shutdown("Critical error has been encountered!");
+            return;
+        }
```

Comment:
jak chcesz to tak robić to wydziel to gdzieś do metody

Reply:
W sumie moge to wrzucić do MessageService, i ta jedynie on tego używa